File: cpp/program/Toy.cpp

#pragma once
#include <iostream>
#include <string>

using namespace std;

class Toy {
private:
    string material;
    string targetAge;
public:
    Toy() {
    }

    Toy(string material, string targetAge) {
        this->material = material;
        this->targetAge = targetAge;
    }

    void setMaterial(string material) {
        this->material = material;
    }

    void setTargetAge(string targetAge) {
        this->targetAge = targetAge;
    }

    string getMaterial() {
        return this->material;
    }

    string getTargetAge() {
        return this->targetAge;
    }

    ~Toy() {
    }
};

File: python/program/Toy.py

class Toy:
    def __init__(self, material="", target_age=""):
        self.__material = material
        self.__target_age = target_age
    
    def set_material(self, material):
        self.__material = material
    
    def set_target_age(self, target_age):
        self.__target_age = target_age
    
    def get_material(self):
        return self.__material
    
    def get_target_age(self):
        return self.__target_age